.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,99 @@
+# Publish archives to PyPI and TestPyPI using GitHub Actions
+name: Publish to PyPI
+
+# Only run for tagged releases
+on:
+  release:
+    types:
+      - released
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build-artifact:
+    name: Build noisepy-seis package
+    runs-on: ubuntu-20.04
+    if: github.repository == 'mdenolle/NoisePy'
```

Comment:
Why is this needed?

Reply:
I do this check, to make sure that this only runs on this repo and not anyone else's fork. This again is probably an overkill since pypi doesn't allow duplicate packages.